### DISCUSSION TITLE
Get rid or warnings during build

### DISCUSSION THREAD

**Discussion by TheZoker** [facebook/docusaurus#4738] [Q&A] [answered]
Hi there,

Currently when I build our project, I get the following warnings:
```
The sidebar file of docs version [current] does not exist. It is optional, but should rather be provided at /jsonforms2-website/sidebars.json
Warn: `docsDir` doesn't exist: "/jsonforms2-website/docs".
Warn: `blogDir` doesn't exist: "/jsonforms2-website/blog".
```

I can't find `docsDir` or `blogDir` anywhere in our config. Also I'm not sure, what the sidebar warning actual means. 
Can someone help to get rid of these warnings? Thanks!

Here is our current config: 
https://github.com/eclipsesource/jsonforms2-website/blob/a833bed/docusaurus.config.js

**Comment by slorber** [comment, marked as answer]
> The sidebar file of docs version [current] does not exist. It is optional, but should rather be provided at /jsonforms2-website/sidebars.json

I think this is due to sidebarsPath not being configured in the FAQ instance. 
Unfortunately, it is currently needed even if it's empty.

Not sure about the 2 other warnings 😅 it does not look like a Docusaurus warning, couldn't find in the source such labels.

**Reply by TheZoker** [reply]
Thanks for your help!

When I set
```js
[
  '@docusaurus/plugin-content-docs',
  {
    id: 'faq',
    path: 'content/faq',
    routeBasePath: 'faq',
    sidebarPath: ' ',
  },
],
```
the issue remains. Do you have another idea how to fix this?

And the two other warnings seem to come from this plugin:
https://github.com/easyops-cn/docusaurus-search-local

So I was able to fix them. But the sidebar warning remains :thinking:

**Reply by slorber** [reply]
You have to use a sidebarPath leading to an actual file. I mean, `sidebars/faq.json` should be empty

**Reply by TheZoker** [reply]
Ah nice, yes that worked. In order to hide the sidebar on the FAQ page, I had to use the following content within `faq.json`:
```js
module.exports = {
};
```

Thanks for the help :smile:

**Reply by slorber** [reply]
great.

Created a bug report because this shouldn't be necessary:
https://github.com/facebook/docusaurus/issues/4752

**Comment by Tobbe** [comment]
This is like one of two results on google for "docusaurus `blogDir` doesn't exist"

The solution for me was to set `indexBlog: false` in the `@easyops-cn/docusaurus-search-local` settings.
(See here for exactly how I solved it https://github.com/cedarjs/cedar/pull/120/files#diff-9dbe126a5a498e56dcf30f25d7fc163f8443517db03458905d3d33303e1cdbf9R105)
